Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -1,37 +1,8 @@
 Write-Verbose "Starting SonarQube PostBuild Step"
 
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
+. ./SonarQubeHelper.ps1
```

Comment:
The file name is very generic and both tasks have files called "SonarQubeHelper.ps1" with different content which is confusing. Something like PreBuildUtils/PreBuildImpl would be remove the possibility for confusion.